ocs_ci/helpers/dr_helpers_ui.py

```
@@ -28,6 +32,665 @@
+
+    for attempt in range(1, 4):
+        try:
+            acm_obj.do_click(
+                cluster_click_locator, use_fallback=False, timeout=timeout
+            )
+            break
+        except (NoSuchElementException, SeleniumTimeoutException):
+            log.info("Zooming out DR Topology view")
+            acm_obj.do_click(topology_loc["zoom_out"])
+            acm_obj.page_has_loaded(retries=3, sleep_time=2)
+            log.info(f"Retry cluster click on DR Topology. attempt {attempt}")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)